connectors/byoc.py

```diff
@@ -178,96 +178,138 @@ async def all_connectors(self):
             yield connector
 
 
-class SyncJob:
-    def __init__(self, elastic_index, connector_id, doc_source=None):
-        self.connector_id = connector_id
-        self.elastic_index = elastic_index
-        self.created_at = datetime.now(timezone.utc)
-        self.completed_at = None
-        self.filtering = Filter()
-        self.client = elastic_index.client
+class ESDocument:
```

Comment:
Should we move that class to `connectors/es/document.py` ?

Reply:
Done in https://github.com/elastic/connectors-python/commit/5d03016809dcc71b19b10fb38df21dbfd79efa46